app/views/layouts/frda.html.erb

```diff
@@ -33,7 +33,7 @@
 
     <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
     <!--[if lt IE 9]>
-      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+      <script src="https://html5shim.googlecode.com/svn/trunk/html5.js"></script>
```

Comment:
This shim file was removed when googlecode was shut down. I think we should just remove the whole thing?